scripts/check-redact-log

```diff
@@ -7,12 +7,12 @@ function error_msg() {
 }
 
 if [[ "$(uname)" == "Darwin" ]] ; then
-   if grep -r -n --color=always --include '*.rs' --exclude hex.rs --exclude-dir target 'encode_upper' . | grep -v log_wrappers ; then
+   if grep -r -n --color=always --include '*.rs' --exclude hex.rs --exclude-dir tikv-ctl --exclude-dir target 'encode_upper' . | grep -v log_wrappers ; then
```

Comment:
no need to change?

Reply:
`make clippy` will complain if this is not changed.